frontend/packages/react-app/src/components/organisms/DownloadEmailCsvPane/index.tsx

```diff
@@ -0,0 +1,138 @@
+/*
+ *     Copyright (C) 2021 TART K.K.
+ *
+ *     This program is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ *
+ *     This program is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+import * as React from "react";
+import { Paper, Typography, Box, Button } from "@material-ui/core";
+import { useState } from "react";
+import { EMAIL_ACTIONS, EmailState } from "../../../reducers/email";
+import { CSVLink } from "react-csv";
+import styled from "styled-components";
+
+export interface DownloadEmailCsvPaneProps {
+  readonly tokenAddress: string;
+  readonly campaignAddress: string;
+  readonly emailState: EmailState;
+  readonly emailDispatch: React.Dispatch<EMAIL_ACTIONS>;
+}
+
+const StyledCSVLink = styled(CSVLink)`
+  text-decoration: none;
+`;
+
+const DownloadEmailCsvPane: React.FC<DownloadEmailCsvPaneProps> = ({
+  tokenAddress,
+  campaignAddress,
+  emailState,
+  emailDispatch,
+}) => {
+  const [isDownloaded, setIsDownloaded] = useState(false);
+  const urlList = emailState.rawTargets.map(
+    (uuid) =>
+      `${window.location.origin}/explore/${tokenAddress}/distributors/${emailState.distributorAddress}/campaigns/${campaignAddress}?uuid=${uuid}`
+  );
+  const emailUrlPair = emailState.emailList.map((email, index) => {
+    return [email, urlList[index]];
+  });
+  const csvData = [["Email", "UUID"], ...emailUrlPair];
```

Comment:
I think Campaign URL, Claim URL, or Unique URL would be easier for users to understand since they don't understand UUID. When sending an email, it is confusing to see UUID.